instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
>  this repo will not have any idea about distances

I don't think we should bring distances per se into this repo, but we could include alarm priorities with a placeholder for the client to integrate the distance. Likely this would be an override as we have already, so only special cases (like end of road high priority) would get a custom text.